app/controllers/ChargeSummaryController.scala

```diff
@@ -237,9 +227,9 @@ class ChargeSummaryController @Inject()(val auth: AuthenticatorPredicate,
 
   def mandatoryViewDataPresent(isLatePaymentCharge: Boolean, documentDetailWithDueDate: DocumentDetailWithDueDate)(implicit user: MtdItUser[_]): Either[ErrorCode, Boolean] = {
 
-    val viewSection1 = isEnabled(ChargeHistory) && (!isLatePaymentCharge && !(isEnabled(CodingOut) && documentDetailWithDueDate.documentDetail.isPayeSelfAssessment))
```

Comment:
Looks like this one would have only been true when coding out was disabled, so would always be false now